src/latex/raw.ml

```diff
@@ -209,10 +209,20 @@ let ocamltag tag pp ppf x = create2 "ocamltag" Fmt.string pp ppf tag x
 
 let math ppf x = Fmt.pf ppf {|$%s$|} x
 
+let remove_empty_last_line x =
```

Comment:
We depend on `astring` in other places, so `String.trim` would work.

Reply:
To trim only on the right, `astring` has `Astring.(String.drop ~rev:true ~sat:Char.Ascii.is_white x)`